test/e2e/check_upgrades.go

```diff
@@ -230,23 +230,19 @@ func getHeight(ctx context.Context, client *http.HTTP, period time.Duration) (in
 	}
 }
 
-func waitForHeight(testnet *testnet.Testnet, node *testnet.Node, height int64, period time.Duration) error {
+func waitForHeight(ctx context.Context, client *http.HTTP, height int64, period time.Duration) error {
 	timer := time.NewTimer(period)
 	ticker := time.NewTicker(100 * time.Millisecond)
 	for {
 		select {
 		case <-timer.C:
```

Comment:
I guess this should also check: `case <-ctx.Done()` in case some cancel happened for the parent context somewhere